addon/plugins/roadsign-regulation-plugin/nodes.ts

```diff
@@ -0,0 +1,87 @@
+import { NodeSpec } from '@lblod/ember-rdfa-editor';
+import {
+  DCT,
+  EXT,
+  MOBILITEIT,
+  PROV,
+} from '@lblod/ember-rdfa-editor-lblod-plugins/utils/constants';
+import { hasRDFaAttribute } from '@lblod/ember-rdfa-editor-lblod-plugins/utils/namespace';
+
+export const roadsign_regulation: NodeSpec = {
+  content: 'block+',
+  group: 'block',
+  attrs: {
+    resourceUri: {},
+    measureUri: {},
+    zonality: {},
+    temporal: {},
+  },
+  toDOM(node) {
+    const { resourceUri, measureUri, zonality, temporal } = node.attrs;
+    return [
+      'div',
+      {
+        property: MOBILITEIT('heeftVerkeersmaatregel'),
+        typeof: MOBILITEIT('Mobiliteitsmaatregel'),
+        resource: resourceUri as string,
+      },
+      [
+        'span',
+        {
+          style: 'display:none;',
+          property: PROV('wasDerivedFrom'),
+          resource: measureUri as string,
+        },
+      ],
+      [
+        'span',
+        {
+          style: 'display:none;',
+          property: EXT('zonality'),
+          resource: zonality as string,
+        },
+      ],
+      [
+        'span',
+        {
+          style: 'display:none;',
+          property: EXT('temporal'),
+          resource: (temporal as string | false).toString(),
+        },
+      ],
+      ['div', { property: DCT('description') }, 0],
+    ];
+  },
+  parseDOM: [
+    {
+      tag: 'div',
+      getAttrs(node: HTMLElement) {
+        if (
```

Comment:
only worry I have here is that we are throwing out any unknown rdfa properties, like rel,rev, etc
probably won't be a practical issue but just in case?